openedx/core/djangoapps/xblock/api.py

```diff
@@ -70,7 +71,7 @@ def get_runtime(user: UserType):
 
 
 def load_block(
-    usage_key: UsageKeyV2,
+    usage_key: UsageKeyV2 | LibraryContainerLocator,
```

Comment:
I'm assuming this is old, and we don't need to support containers in `load_block` anymore ?

We can remove `get_container_block` and `library_container_xml` and a bunch of other changes.

Reply:
@bradenmacdonald We still need to load unit xblock (in course they are xblocks) so it was easier to just update this method.

Reply:
I'm not sure I follow. In the course, the Unit containers will have been converted to `vertical` XBlocks, and can be loaded via modulestore APIs like `get_item()`